prow/plugins/trigger/generic-comment.go

```diff
@@ -87,6 +95,24 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 		}
 	}
 
+	pr, err := refGetter.PullRequest()
+	if err != nil {
+		return err
+	}
+
+	// Process "help" comments
+	if testHelpRe.MatchString(gc.Body) {
+		available, err := availablePresubmits(c.GitHubClient, gc.Body, org, repo, pr.Base.Ref, pr.Number, presubmits, c.Logger)
```

Comment:
I think we should also fix https://github.com/kubernetes/test-infra/issues/13059 in this PR by responding with the same comment if someone requested an invalid test

Reply:
Thank you for the review. Fixed. I included `/test` with unrecognised target, `/test` without any targets, and `/retest` with trailing words in the fix to address all cases listed on #13059